integration-tests/features/stack_analyses_v2_minimal.feature

```diff
@@ -34,6 +34,8 @@ Feature: Stack analysis v2 API Minimal
     Then I should find that alternate components replace user components
     Then I should find the security node for all dependencies
     Then I should find the security node for all alternate components
+    When I wait 60 seconds
+    Then I should find matching topic lists for all user_stack_info/analyzed_dependencies components
```

Comment:
I know that one minute might be 'safe number', but is not it better to perform additional test (in a loop+sleep) that the requested node has been added to the stack analysis?

Reply:
@tisnik  I think wait time is not required. I will add another step to verify the prerequisite for this test step.